component/site/views/flat/assets/css/events_css.css

```diff
@@ -999,4 +999,7 @@ div#jumpto form {text-align:center; width: 93%;margin-top:10px;}
     width: 195px !important;
     text-align: left;
     display: block;
-}
\ No newline at end of file
+}
+
+/** Make sure choosen list is under input **/
+#jevents .chzn-container.chzn-container-single .chzn-results {clear:both;}
```

Comment:
See:
https://www.dropbox.com/s/ppumj9ya6s1ecix/Screenshot%202015-09-07%2015.13.38.jpg?dl=0

Adding this CSS tells it to clear and drop to the next line and fixes the display issue.